test/inductor/test_torchinductor.py

```diff
@@ -499,30 +477,29 @@ def downcast_fn(x):
 
 
 def _run_and_assert_no_indirect_indexing(test_case, func, *args, **kwargs):
-    result, source_codes = run_and_get_code(func, *args, **kwargs)
-
-    for code in source_codes:
-        for line in code.split("\n"):
-            stmt = None
-            # Find indexing expressions
-            if ".load(" in line:
-                stmt = line.split(".load")[-1]
-            elif "tl.store" in line:
-                stmt = line.split(".store")[-1]
-                stmt = ",".join(stmt.split(",")[:-2])  # Remove store value and mask
-            elif ".store" in line:
-                stmt = line.split(".store")[-1]
-            elif "[" in line:
-                stmt = line.split("[")[-1].split("]")[0]
-
-            if stmt is None:
-                continue
-
-            # indirect indexing involves a `tmp` variable
-            test_case.assertTrue(
-                "tmp" not in stmt,
-                msg=f"Found indirect indexing in statement '{stmt}' from code:\n{code}",
-            )
+    result, code = run_and_get_code(func, *args, **kwargs)
```

Comment:
Okay, I can switch to `run_and_get_multiple_code` if you would like to keep git blame. I made this change because my initial attempt didn't have that `multiple_code` function, and this test doesn't actually need to split up the source code by file.

Reply:
nevermind - i didnt realize there was real changes. thought it was just variable name rename. changes are fine then